src/services/requiredSignoffs.js

```diff
@@ -5,9 +5,26 @@ const getRequiredSignoffs = objectName =>
   axios.get(`${BASE_URL}/${objectName}`);
 const getScheduledChanges = objectName =>
   axios.get(`${BASE_URL}/scheduled_changes/${objectName}`);
+const updateRequiredSignoff = params => {
+  const { useScheduledChange, scId, ...postData } = params;
+  const type = postData.channel ? 'product' : 'permissions';
+  const scPath = scId ? `/${scId}` : '';
+  const url = useScheduledChange
+    ? `${BASE_URL}/scheduled_changes/required_signoffs/${type}${scPath}`
+    : `${BASE_URL}/required_signoffs/${type}`;
+  // TODO: where can we set this globally as a default?
```

Comment:
For views, we expose the `withUser` utility that passes the user object as a prop. For utilities, one way to achieve this here is to use the `withUser` in `ViewSignoff/index.jsx ` and pass the user object as an option.

```jsx
const updateRequiredSignoff = (params, options = {}) => {
  const { user } = options;

  // ...
});
```

Another possible way (perhaps even neater) would be to create a `utils/axios.js` that sends the access token whenever the user is logged in and we  are sending a request to our own backend. We would use this new utility as the axios object.

Reply:
Something would wrap the `axios` object (or maybe each method we use?) and use `transformRequest` to set the header?